### PR TITLE
add op stack walkthrough to docs

### LINKED ISSUE #1705: docs: add op stack walkthrough to docs
https://www.youtube.com/watch?v=lOLw4uLX644

### PR DESCRIPTION
**Pull request by Mackenzie-OO7**
closes #1705

## Summary by CodeRabbit

- **New Features**
	- Enhanced guide for running an OP Stack rollup with Celestia, including detailed sections on mock L1 environment and real L1 testnet deployment.
	- Added instructions for observing data submissions and account activity.
	- Included setup tips for macOS users and expanded deployment instructions with relevant links.
	- Added a link to a YouTube video for visual guidance.

- **Documentation**
	- Updated configuration requirements for testnet setup, including chain ID, RPC URL, and deployment keys.
	- Clarified configuration example for deploying to a live EVM network, including specific parameters for the rollup and Sepolia Ethereum.

### CODE REVIEW

File: how-to-guides/optimism.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ import constants from '/.vitepress/constants/constants.js'
 
 This guide will show you how to run your own OP Stack devnet and testnet that posts data to Celestia's Mocha testnet using [roll-op](https://github.com/celestiaorg/roll-op) and [op-plasma-celestia](https://github.com/celestiaorg/op-plasma-celestia).
 
+The roll-op tool is used to deploy and manage the OP Stack rollup environment, including the rollup, batcher, and other components. While the op-plasma-celestia integration allows the OP Stack to utilize Celestia's Mocha testnet as the data availability (DA) layer.
```
Comment (jcstein) [inline how-to-guides/optimism.md:14]:
```suggestion
The roll-op tool is used to deploy and manage the OP Stack rollup environment, including the rollup, batcher, and other components. While the op-plasma-celestia integration allows the OP Stack to use Celestia as the data availability (DA) layer.
```

File: how-to-guides/optimism.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ import constants from '/.vitepress/constants/constants.js'
...
+
+This guide is in two parts:
+
+- First, you'll spin up a mock L1 environment and deploy a devnet that posts data to the Mocha testnet. 
+
```
Comment (jcstein) [inline how-to-guides/optimism.md:19]:
```suggestion
```

**Review by jcstein — CHANGES_REQUESTED** [review]
lgtm, but should include the youtube link

**Comment by Mackenzie-OO7** [comment]
> lgtm, but should include the youtube link

done! thanks for reviewing

**Review by jcstein — APPROVED** [review]

**Pull request merged**